lib/api/src/grpc/validate.rs

```diff
@@ -40,19 +40,6 @@ where
     }
 }
 
-impl<V> ValidateExt for Vec<V>
```

Comment:
An implement for `Vec<V>` is now in the library creating conflicts.
https://github.com/Keats/validator/blob/0872516333bf365668a22e522c57dac64e60d398/validator/src/traits.rs#L50

Reply:
This is fantastic. This implementation has caused some hidden panics before. :smile: